addons/sourcemod/scripting/szf/dhook.sp

```diff
@@ -73,11 +73,16 @@ void DHook_HookGamerules()
 	DHookGamerules(g_hDHookRoundRespawn, false, _, DHook_RoundRespawnPre);
 }
 
-public MRESReturn DHook_DeactivatePre(int iThis, Handle hParams)
+public MRESReturn DHook_CGameUI_Deactivate_Pre(int iThis, Handle hParams)
 {
 	if (!g_bEnabled)
 		return MRES_Ignored;
 	
+	int iClient = GetEntPropEnt(iThis, Prop_Data, "m_player");
+	// Don't allow zombies drop ammo and dropped weapon
+	if (0 < iClient <= MaxClients && IsZombie(iClient))
+		return MRES_Supercede;
+	
```

Comment:
I should've said it more, commit https://github.com/redsunservers/SuperZombieFortress/commit/6bf3a616d0b9276cf5b217d6ee4c3c9cc276b06f was an attempt to allow survivors drop and pickup dropped weapon, but realized it would take a lot more work so i reverted it at commit https://github.com/redsunservers/SuperZombieFortress/commit/b22e88c254a79b9c4b3347fc10e26f729bd19e17. `DHook_DropAmmoPackPre` function was also meant to be removed but i missed that, and was accidentally merged into `DHook_CGameUI_Deactivate_Pre`.

Reply:
In that case, it's better to close the PR yeah. Since it's a just one line change, that will look weird in git history otherwise lol.